user-interface/7003-MORD-manage_orders.md

```diff
@@ -1 +1,202 @@
+- **must** see [order type](9001-DATA-data_display.md#order-type) (<a name="7003-MORD-005" href="#7003-MORD-005">7003-MORD-005</a>)
+- if order created by [pegged or liquidity provision shape](9001-DATA-data_display.md#order-origin): **should** see order origin
+  - **could** see what part of the liquidity shape or pegged order shape this relates to or it's offset+reference See [pegged orders](#pegged-order-shapes) and [liquidity provisions](#liquidity-order-shapes) shapes below.
+  - **could** see link to full shape
+
+- **should** see how much of the order's [size](9001-DATA-data_display.md#size) has been filled e.g. if the order was for `50` but so far only 10 have traded I should see Filled = `10`. Note: this is marked as a should because in the case of Rejected order and some other scenarios it isn't relevant.
+- **should** see how much of the order's [size](9001-DATA-data_display.md#size) remains. 
+
+- if order type = `Limit`: **must** see the Limit [price](9001-DATA-data_display.md#quote-price) that was set on the order
+- if order type = `Market`: **must** not see a price for active or parked orders, a `-`, `Market` or `n/a` is more appropriate (API may return `0`).
+
+- **must** see the [time in force](9001-DATA-data_display.md#time-in-force) applied to the order (can be abbreviated here) (<a name="7003-MORD-006" href="#7003-MORD-006">7003-MORD-006</a>)
+- **should** see "created at" [time](9001-DATA-data_display.md#time)
+- **could** see updated at (this is used by the system when an order is amended, or repriced (in pegged and LP) not sure this in needed)
+
+- **should** see time priority (how many orders are before mine at this price)
+  
+- if the order is `Active` &amp; **not** part of a liquidity or peg shape: **must** see an option to [amend](#amend-order---price) the individual order (<a name="7003-MORD-007" href="#7003-MORD-007">7003-MORD-007</a>)
+- if the order is `Active` &amp; is part of a liquidity or peg shape: **must** **not** see an option to amend the individual order (<a name="7003-MORD-008" href="#7003-MORD-008">7003-MORD-008</a>)
+  - **could** see a link to amend shape
+- if the order is `Active` &amp; **not** part of a liquidity or peg shape: **must** see an option to [cancel](#cancel-orders) the individual order
+- if the order is `Active` &amp; is part of a liquidity or peg shape: **must** **not** see an option to cancel the individual order
+  - **could** see a link to cancel shape
+
+... so I can understand the state of my orders and decide what to do next
+### Filters
+
+When looking at a list of orders, I...
+
+- **should** have the ability to see all orders regardless of status in one list
+- **should** have the ability to see only active &amp; parked orders TODO update this based on what happens to parked orders
```

Comment:
TODO needs updating

Reply:
resolved in : https://github.com/vegaprotocol/specs/pull/1334/commits/01f208d055c69bb4b89a6e4cb0b84d655259886e should be able to filter by parked and active